src/git-kill

```diff
@@ -13,10 +13,11 @@ function branch_to_kill {
 
 function preconditions {
   target_branch=$(branch_to_kill "$@")
-  parent_branch_name=$(parent_branch "$target_branch")
-
   ensure_is_feature_branch "$target_branch" "You can only kill feature branches."
 
+  ensure_knows_parent_branches "$target_branch"
```

Comment:
The spec sounds like it applies to all commands, yet the implementation is only for git-kill. Are you planning to add scenarios for the other commands later? Or do we want to move the spec into the folder for git-kill?